src/components/common/DetailsList.vue

```diff
@@ -21,7 +21,7 @@
       <v-list-item class="pl-0 pr-0 detail-body" v-for="(comment, index) in filing.comments" :key="index">
         <v-list-item-content>
           <v-list-item-title class="body-2">
-            <strong v-if="!isRoleStaff">Registry Staff</strong>
+            <strong v-if="!isRoleStaff">BC Registries Staff</strong>
```

Comment:
This is "who" made this detail comment.

(Unfortunately the staff IDIR is still provided in the API response 🤦‍♂️ - I create [10449](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/10449) for that, and at the same time, the API can change "Registry Staff" -> "BC Registries Staff".)

Reply:
this works.